### ISSUE TITLE
IP output spectra files are not correctly parsed by YamboFile for yambo==5.1.0

### ISSUE THREAD

**Issue by mikibonacci** [yambo-code/yambopy#51]
Hi @palful 

This happens to me in yambo 5.1.0 and yambopy master branch.  
The structure of the file is:

```bash
#    E[1] [eV]          Im(eps)            Re(eps)
#
        0.000000           0.538996           8.660297
        0.010010           0.539213           8.660860
```

I am currently working on this in aiida-parsing branch of my fork, the plan is to get this merged in the master of my fork (so we also align the master branch and we can get rid of the aiida-parsing).  I will open a PR but in draft state (should not be urgent).

The problem should be in the tags regex

We can discuss more during the total energy, if you are there :)  thanks!

**Comment by palful** [comment]
Ciao @mikibonacci, interesting, I think we had only one patch to YamboFile in years via PR #41 . Or maybe it could be that yambo 5.1.0 was never used much with YamboFile and this slipped under the radar?

**Comment by mikibonacci** [comment]
Yeah, can be. Maybe the IP alone is just so uncommon case that it was just ignored
